geopandas/sindex.py

```diff
@@ -168,18 +168,39 @@ def query_bulk(self, geometry, predicate=None, sort=False):
         """
         raise NotImplementedError
 
-    def nearest(self, geometry):
+    def nearest(
+        self, geometry, return_all=True, max_distance=None, return_distance=False
+    ):
         """
-        Returns the nearest geometry to the input geometries.
+        Return the nearest geometry in the tree for each input geometry in
+        ``geometry``.
 
         .. note::
             ``nearest`` currently only works with PyGEOS >= 0.10.
 
-        Note that if pygeos is not available, geopandas
-        will use rtree for the spatial index, where nearest temporarily has a different
-        function signature to temporarily preserve existing functionality. See the
-        documentation of :meth:`rtree.index.Index.nearest` for the details on the
-        ``rtree``-based implementation.
+            Note that if PyGEOS is not available, geopandas will use rtree
+            for the spatial index, where nearest temporarily has a different
+            function signature to temporarily preserve existing
```

Comment:
```suggestion
            for the spatial index, where nearest has a different
            function signature to temporarily preserve existing
```

Duplicated `temporarily` (from the original docstring).